### PR TITLE
fix: Update to latest yargs to fix security issue with the subdependency "mem"

### PR DESCRIPTION
**Pull request by Xotic750**
Drops support for node 4

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 language: node_js
+cache:
+  npm: false
 notifications:
   email: false
 node_js:
   - '10'
   - '8'
   - '6'
-  - '4'
```
Comment (ljharb) [inline .travis.yml]: dropping support for node 4 is not acceptable, i'm afraid.

Comment (Xotic750) [reply .travis.yml]: Ok, that is yargs https://github.com/yargs/yargs/blob/master/package.json#L77

Comment (ljharb) [reply .travis.yml]: yargs v11 supports node 4.

Comment (Xotic750) [reply .travis.yml]: But unfortunately does not fix "mem"

Comment (ljharb) [reply .travis.yml]: then this isn't a fixable non-problem, and you'll just have to ignore the warning.

Comment (Xotic750) [reply .travis.yml]: Ah well, thanks.

**Comment by ronkorving** [comment]
May I ask why Node 4 support (which has been end-of-life since 2018-04-30) is so important?

**Pull request closed without merging**